scripts/mkcloud

```diff
@@ -164,6 +164,14 @@ host_mtu=$(determine_mtu)
 iscloudver 7plus && [[ $arch == x86_64 ]] && \
     [[ -n $want_efi ]] && firmware_type="uefi"
 
+#external ceph cluster
+: ${want_external_ceph:=0}
```

Comment:
move the `want_external_ceph` default into mkcloud-common so that it is also used for qa_crowbarsetup